lib/Agrammon/Web/Service.pm6

```diff
@@ -273,8 +273,8 @@ class Agrammon::Web::Service {
         }
     }
 
-    method store-input-comment(Agrammon::Web::SessionUser $user, :$dataset!, :$variable!, :$comment --> Nil) {
-        Agrammon::DB::Dataset.new(:$user, :name($dataset)).store-input-comment(:$variable, :$comment);
+    method store-input-comment(Agrammon::Web::SessionUser $user, $dataset!, $variable!, $comment --> Nil) {
```

Comment:
Positional parameters are required by default

Reply:
Fixed. Those were none-positional arguments before and again I didn't pay attention to Comma.